components/roadmap-grid/RoadmapDetailedView.tsx

```diff
@@ -145,28 +147,38 @@ export function RoadmapDetailed({
   return (
     <>
       {isDevMode && <NumSlider msg="how many header ticks" value={numHeaderTicks} min={5} max={60} setValue={setNumHeaderTicks}/>}
-      {isDevMode && <NumSlider msg="how many grid columns" value={numGridCols} min={20} max={60} step={numHeaderTicks} setValue={setNumGridCols}/>}
-
-      <Box className={`${styles.timelineBox} ${ viewMode=='detail' ? styles.detailView : '' }`} >
-        <Grid ticksLength={numGridCols}>
-          {ticksHeader.map((tick, index) => (
-
-            <GridHeader key={index} tick={tick} index={index} numHeaderTicks={numHeaderTicks} numGridCols={numGridCols}/>
-          ))}
+      {isDevMode && <NumSlider msg="how many grid columns" value={numGridCols} min={20} max={60} step={numHeaderTicks} setValue={setNumGridCols} />}
+
+      {globalLoadingState.get() ? (
+        <Stack pt={"20px"} hidden={!globalLoadingState.get()}>
+          <Skeleton isLoaded={!globalLoadingState.get()} height='60px' />
+          <Skeleton isLoaded={!globalLoadingState.get()} height='150px' />
+          <Skeleton isLoaded={!globalLoadingState.get()} height='150px' />
+          <Skeleton isLoaded={!globalLoadingState.get()} height='150px' />
+          <Skeleton isLoaded={!globalLoadingState.get()} height='150px' />
+        </Stack>
```

Comment:
I'd rather return early on a guard class than have a huge ternary that's difficult to parse